example/src/App.tsx

```diff
@@ -88,7 +89,7 @@ const RenderListItem = React.memo(
               {isLoading ? (
                 <ActivityIndicator color={'#FF0000'} />
               ) : (
-                <Image
+                <FastImage
```

Comment:
> tintColor prop and not in style

I tried, but it fails on iOS; several posts about it exist. I will try to find them again and post them here or retry!

Reply:
@dprevost-LMI  Thank you for checking this out. This button is working for me in the simulator as well as the device too.
Sim iOS: 17.5
device OS: 16.4

Can you check if issue only exists with simulator or that specific version.

Reply:
😮  Not working on simulator with iOS 17.0. I will try with iOS 17.5, but I cannot try with a physical phone since I need the certificate/signing capabilities
<img width="733" alt="image" src="https://github.com/user-attachments/assets/c7b5e687-99ca-4b06-9437-4b1fd9630f7f">

Reply:
Same on 17.5 with iPhone 15 Pro max

<img width="981" alt="image" src="https://github.com/user-attachments/assets/2be5bf2f-d805-4a96-9b8d-39daaa3837d3">

Reply:
To make it work, I need to remove the scale. Commenting and uncommenting the scale also "fixes" the issue, but I'm not sure I really understand what is going on there! Using FastImage seems a simple and quick fix, what do you think?

https://github.com/user-attachments/assets/1ce00fb1-bf55-452b-b8aa-402a26aa7516

Reply:
Yes, I don't think we need to go into that hassle to find out what is happening here. We can use Fast Image if this is a fixing issue.